README.md

```diff
@@ -26,22 +26,22 @@ After configuring `quarkus BOM`:
 </dependencyManagement>
 ```
 
-You can just configure the `quarkus-minio` extension by adding the following dependency:
+You can just configure the `quarkiverse-minio` extension by adding the following dependency:
 
 ```xml
 <dependency>
     <groupId>io.quarkiverse.minio</groupId>
-    <artifactId>quarkus-minio-client</artifactId>
+    <artifactId>quarkiverse-minio</artifactId>
     <version>${latest.release.version}</version>
 </dependency>
 ```
 <!--
-***NOTE:*** You can bootstrap a new application quickly by using [code.quarkus.io](https://code.quarkus.io) and choosing `quarkus-minio`.
+***NOTE:*** You can bootstrap a new application quickly by using [code.quarkus.io](https://code.quarkus.io) and choosing `quark-minio`.
```

Comment:
```suggestion
***NOTE:*** You can bootstrap a new application quickly by using [code.quarkus.io](https://code.quarkus.io) and choosing `quarkiverse-minio`.
```

Reply:
In fact, as a follow to a chat on zulip. This extension isn't part of the platform, so it can't be found on code.quarkus.io. Not as of now any, so I am thinking about removing those...

Reply:
Ah yes, good idea. I was just looking at the typo here ;)

We're working on that and should be available soon

Reply:
Removed.